vector/src/main/java/im/vector/app/features/settings/devices/v2/DevicesViewModel.kt

```diff
@@ -63,6 +69,14 @@ class DevicesViewModel @AssistedInject constructor(
         observeDevices()
         refreshDevicesOnCryptoDevicesChange()
         refreshDeviceList()
+        refreshIpAddressVisibility()
+    }
+
+    private fun refreshIpAddressVisibility() {
+        val shouldShowIpAddress = sharedPreferences.getBoolean(VectorPreferences.SETTINGS_SESSION_MANAGER_SHOW_IP_ADDRESS, false)
```

Comment:
I think we should either define the default value for this preference somewhere in VectorPreferences or create a method to retrieve this setting so that it can be reused accross the screens.

Reply:
Good idea, done.